src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let res;
+
+  if (days < 3) {
+    res = days * 40;
+  };
+
+  if ((days >= 3) && (days < 7)) {
+    res = days * 40 - 20;
+  };
+
+  if (days >= 7) {
+    res = days * 40 - 50;
+  };
```

Comment:
делай return сразу в if

Reply:
```suggestion
  if (days < 3) {
    return days * 40;
  };

  if ((days >= 3) && (days < 7)) {
    return (days * 40) - 20;
  };

    return (days * 40) - 50;
```
И последний if не нужен будет 
Так же расставляй скобочки, чтобы легко читалось какая операция за какой последует

Reply:
повыноси числа(20,40,50,3,7) в переменные, и дай им осмысленное название